<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0.689, y1:0.233, x2:0.578, y2:0.136273, stop:0.0945274 rgba(83, 79, 20, 255), stop:1 rgba(255, 255, 255, 255))}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Circle_Shape">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 100; 
background-color: rgb(0, 255, 72);
                              border : 2px solid black</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Circle_Radius">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Circle_Label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Enter radius</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Circle_Submit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
border-radius : 15; 
background-color: rgb(255, 233, 188);
                              border : 2px solid black
</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
